macros/macros.go

```diff
@@ -17,6 +17,9 @@ type EndpointTemplateParams struct {
 	GvlID       string
 	PageID      string
 	SupplyId    string
+	SspID       string // Ownadx adapter Macros.
+	SeatID      string
+	TokenID     string
```

Comment:
The tests are failing because you need to add testing defaults for these new values.

Reply:
Adding Testing defualt for this new values, please verify.